### PR TITLE
Complex e2e tests

### PR DESCRIPTION
**Pull request by savelyev-an**
add tests for running system: 
1) kafka etc
2) vault (flant_iam + flant_iam_auth)
3) client (authd+cli)
4) server (authd+server-accessd)

Tests 
1) create tenant, project, user etc at vault
2) check binary exists at client and server
3) configure client and server (configs for authd etc)
4) run authd and server-access at server and check appearance of valid authorized_keys

### CODE REVIEW

**Review by tuchinsky — APPROVED** [review]

File: vault-plugins/flant_iam/backend/tests/specs/helpers.go [hunk 1]
```diff
@@ -136,16 +152,95 @@ func CreateRandomRole(roleAPI api.TestAPI) model.Role {
...
+		"tenant": rb.TenantUUID,
+	}
+	bytes, _ := json.Marshal(rb)
+	var createPayload map[string]interface{}
+	json.Unmarshal(bytes, &createPayload) //nolint:errcheck
+	createData := rolebindingAPI.Create(params, url.Values{}, createPayload)
+	rawRoleBinding := createData.Get("role_binding")
+	data := []byte(rawRoleBinding.String())
+	var roleBinding model.RoleBinding
+	err := json.Unmarshal(data, &roleBinding) //nolint:errcheck
```
Comment (shvgn) [inline vault-plugins/flant_iam/backend/tests/specs/helpers.go:178]: `//nolint:errcheck` is redundant here. +3 times below.

File: vault-plugins/flant_iam/backend/tests/specs/tenant_featureflag/spec.go [hunk 2]
```diff
@@ -13,17 +13,17 @@ import (
 )
 
 var (
-	TestTenantAPI      api.TestAPI
-	TestFeatureFlagAPI api.TestAPI
-	TestTenantFFApi    api.TestAPI
+	TenantAPI      api.TestAPI
+	FeatureFlagAPI api.TestAPI
+	TestApi        api.TestAPI
```
Comment (shvgn) [inline vault-plugins/flant_iam/backend/tests/specs/tenant_featureflag/spec.go:18]: I guess, `TestApi` should be `TestAPI` as it is elsewhere

File: e2e/tests/crud/tenant_featureflag/api_test.go [hunk 3]
```diff
@@ -12,9 +12,9 @@ import (
 
 func Test_tenantFeatureFlagCRUD(t *testing.T) {
 	rootClient := lib.NewConfiguredIamVaultClient()
-	tenant_featureflag.TestTenantAPI = lib.NewTenantAPI(rootClient)
-	tenant_featureflag.TestFeatureFlagAPI = lib.NewFeatureFlagAPI(rootClient)
-	tenant_featureflag.TestTenantFFApi = lib.NewTenantFeatureFlagAPI(rootClient)
+	tenant_featureflag.TenantAPI = lib.NewTenantAPI(rootClient)
+	tenant_featureflag.FeatureFlagAPI = lib.NewFeatureFlagAPI(rootClient)
+	tenant_featureflag.TestApi = lib.NewTenantFeatureFlagAPI(rootClient)
```
Comment (shvgn) [inline e2e/tests/crud/tenant_featureflag/api_test.go:17]: TestApi -> TestAPI, I guess

**Review by shvgn — APPROVED** [review]
All tests pass, I suggest doing only cosmetic changes

File: vault-plugins/flant_iam/backend/tests/specs/helpers.go (hunk 1 above)
Comment (savelyev-an) [reply vault-plugins/flant_iam/backend/tests/specs/helpers.go]: fixed

**Pull request merged**